src/components/MainView/MainView.jsx

```diff
@@ -1,11 +1,22 @@
 import { useState, useEffect } from "react";
 import { MovieCard } from "../MovieCard/MovieCard";
 import { MovieView } from "../MovieView/MovieView";
+import { LoginView } from "../LoginView/LoginView";
+import { SignupView } from "../SignUpView/SignUpView";
 
 export const MainView = () => {
+  const storedUser = JSON.parse(localStorage.getItem("user"));
+  const storedToken = localStorage.getItem("token");
+  const [user, setUser] = useState(storedUser? storedUser : null);
```

Comment:
Just the default value to null as your not using type.

Reply:
I do not understand this one.